eng/Version.Details.xml

```diff
@@ -0,0 +1,12 @@
+<Dependencies>
```

Comment:
I think @jonpryor wasn't thrilled about a new `eng` folder.

Is there any way to put these in `build-tools`? If not, we probably have to put them here.

dotnet/maui already has an `eng` folder, so it will be fine.

Reply:
I haven't found a way to provide alternate paths for files to `darc`, so unless we find the source and PR into that tool to check multiple paths we'll have to keep `eng`.

Reply:
There is this `$(RepositoryEngineeringDir)`:

https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/src/Microsoft.DotNet.Arcade.Sdk/tools/SourceBuild/SourceBuildArcadeTools.targets#L20

https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L44

But if we put our own value for `$(RepositoryEngineeringDir)` in `Directory.Build.props`, I don't know if the tooling would know to parse that value and use the correct value for `$(RepositoryEngineeringDir)`.

Reply:
I think we'd need to PR something here (and perhaps elsewhere) - https://github.com/dotnet/arcade-services/blob/bb98f0d13ac51eb81131b5ef382ada12fd84e8e0/src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/VersionFiles.cs#L16